frontend/src/System/Tasks/Queued/QueuedTaskRow.tsx

```diff
@@ -41,14 +41,14 @@ function getStatusIconProps(status: string, message: string | undefined) {
         name: icons.CHECK,
         kind: kinds.SUCCESS,
         title: message === 'Completed' ? title : `${title}: ${message}`,
-      };
+      } as const;
```

Comment:
Typing the function returning this works the same if you prefer this syntax

Reply:
Creating a type for what is being returned here would be preferred.